jupyter_book/book_template/scripts/clean.py

```diff
@@ -1,5 +1,4 @@
-"""A helper script to "clean up" all of your notebooks and
-generated markdown files."""
+"""A helper script to "clean up" all of your generated markdown, zip, and HTML files."""
```

Comment:
I had updated this comment, since at the time I was 'cleaning' out created zip files, but as is, the reference to zip files is outdated.